about/static/about.css

```diff
@@ -112,3 +114,29 @@ a.personal-link span.overlay {
 a.personal-link span.overlay:hover {
     box-shadow: 0px 0px 5px gray;
 }
+
+@media (max-width: 991px) {
+  div.member-info {
```

Comment:
I really don't like how you've set the height manually. I don't know why you guys weren't seeing this, but in my browser the javascript I wrote was making everything the same size correctly. We should've spent more time on a javascript solution which is far more flexible, rather than manually hard-coding the desired heights.